TombEngine/Scripting/Internal/TEN/Objects/Static/StaticObject.cpp

```diff
@@ -30,28 +31,39 @@ namespace TEN::Scripting
 			sol::meta_function::index, IndexError,
 			sol::meta_function::new_index, NewIndexError,
 
+			// Getters
 			ScriptReserved_StaticGetName, &Static::GetName,
-			ScriptReserved_StaticGetSlot, &Static::GetSlot,
+			ScriptReserved_StaticGetObjectId, &Static::GetObjectId,
```

Comment:
While I agree that previous name `GetSlot` was confusing, `GetObjectID` name is also misleading, for the reason I described above (static meshes just have numerical slot but not object ID).